dspace/config/crosswalks/oai/metadataFormats/oai_openaire.xsl

```diff
@@ -476,30 +485,91 @@
+    <xsl:template match="doc:element[@name='issn']" mode="datacite_ids">
+        <xsl:call-template name="relatedIdentifierTemplate">
+            <xsl:with-param name="value" select="./doc:element/doc:field[@name='value']/text()"/>
+            <xsl:with-param name="relatedIdentifierType" select="'ISSN'"/>
+            <xsl:with-param name="relationType" select="'IsPartOf'"/>
+        </xsl:call-template>
+    </xsl:template>
+
+	<!-- handle: dc.identifier.ismn -->
+    <xsl:template match="doc:element[@name='ismn']" mode="datacite_ids">
+        <xsl:call-template name="relatedIdentifierTemplate">
+            <xsl:with-param name="value"
+                select="concat('ISMN:',normalize-space(./doc:element/doc:field[@name='value']/text()))"/>
+            <xsl:with-param name="relatedIdentifierType" select="'URN'"/>
+            <xsl:with-param name="relationType" select="'IsPartOf'"/>
+        </xsl:call-template>
+    </xsl:template>
+
+	<!-- handle: dc.identifier.govdoc -->
+    <xsl:template match="doc:element[@name='govdoc']" mode="datacite_ids">
+        <xsl:call-template name="relatedIdentifierTemplate">
+            <xsl:with-param name="value"
+                select="concat('govdoc:',normalize-space(./doc:element/doc:field[@name='value']/text()))"/>
+            <xsl:with-param name="relatedIdentifierType" select="'URN'"/>
+            <xsl:with-param name="relationType" select="'IsPartOf'"/>
+        </xsl:call-template>
+    </xsl:template>
+
+	<!-- handle: dc.identifier.isbn -->
+    <xsl:template match="doc:element[@name='isbn']" mode="datacite_ids">
+        <xsl:call-template name="relatedIdentifierTemplate">
+            <xsl:with-param name="value" select="./doc:element/doc:field[@name='value']/text()"/>
+            <xsl:with-param name="relatedIdentifierType" select="'ISBN'"/>
+            <xsl:with-param name="relationType" select="'IsPartOf'"/>
+        </xsl:call-template>
+    </xsl:template>
+
+	<!-- handle: dc.identifier.sici -->
+    <xsl:template match="doc:element[@name='sici']" mode="datacite_ids">
+        <xsl:call-template name="relatedIdentifierTemplate">
+            <xsl:with-param name="value"
+                select="concat('sici:',normalize-space(./doc:element/doc:field[@name='value']/text()))"/>
+            <xsl:with-param name="relatedIdentifierType" select="'URN'"/>
+            <xsl:with-param name="relationType" select="'IsPartOf'"/>
+        </xsl:call-template>
+    </xsl:template>
+
+	<!-- handle: dc.identifier.other -->
+    <xsl:template match="doc:element[@name='other']" mode="datacite_ids">
+        <xsl:call-template name="relatedIdentifierTemplate">
+            <xsl:with-param name="value" select="./doc:element/doc:field[@name='value']/text()"/>
+            <xsl:with-param name="relatedIdentifierType" select="'URN'"/>
+            <xsl:with-param name="relationType" select="'IsPartOf'"/>
+        </xsl:call-template>
+    </xsl:template>	
+
+	<!-- handle: dc.identifier.doi -->
```

Comment:
This isn't an out-of-the-box DSpace field.   We should either note that here, or perhaps comment out this template.  If sites choose to add this field, they can uncomment this template to update the OAI-PMH configuration.

Reply:
There are also cases where it can be treated as an alternateIdentifier if manually assigned from outside DSpace. Perhaps it is best to leave it up to the repository managers.

Reply:
@tdonohue there is also other transformations in place supporting `dc:identifier.doi` like `uketd_dc`:
https://github.com/DSpace/DSpace/blob/main/dspace/config/crosswalks/oai/metadataFormats/uketd_dc.xsl#L221-L223

I don't see any major implications for leaving it. 
But I think to remove it from here, for consistency purposes, all other existing references to `dc:identifier.doi` should also be removed. What do you think?

Reply:
@paulo-graca : Good catch, I was not aware this had been done previously as well.  I'm OK then with letting this in, if we feel there are users who would make use of "dc.identifier.doi".   It just seems a bit odd to add in logic not relating to an out-of-the-box DSpace metadata field...though maybe it's a field we should be considering.

In any case, I'm OK with leaving this in, as you've proven we've done so previously.